src/services/env/Env.ts

```diff
@@ -85,3 +90,19 @@ export function semver(version: string): { major: string, minor: string, patch:
     pre: `${major}.${minor}.${patch}${pre !== undefined ? `-${pre}` : ''}`,
   }
 }
+
+/**
+ * @returns an absolute URL given the current origin and a base URL or base path string. When a path is provided, it is concatenated to the current origin; when a URL is provided, no concatenation takes place. In both cases, the URL is returned without any trailing slashes.
+ */
+function getAbsoluteUrl(origin: string, baseUrlOrPath: string): string {
```

Comment:
Maybe this should be tested?

Reply:
I’ve added tests for this in https://github.com/kumahq/kuma-gui/pull/685/commits/e8bf1314182c3bbda787cd5b4d6129be28ca32c0.